yelp_package/dockerfiles/hacheck-sidecar/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM docker-dev.yelpcorp.com/xenial_yelp
+FROM docker-dev.yelpcorp.com/bionic_yelp
+ARG HACHECK_VERSION=0.18.2-yelp1
 
 ARG PIP_INDEX_URL=https://pypi.yelpcorp.com/simple
 ENV PIP_INDEX_URL=$PIP_INDEX_URL
 
-RUN apt-get update && apt-get install -y hacheck python paasta-tools=0.97.72-yelp1
-RUN mkdir -p /etc/paasta
+RUN apt-get update && \
+    apt-get install -y hacheck=${HACHECK_VERSION} paasta-tools=0.97.72-yelp1 && \
```

Comment:
should we bump paasta-tools here? (or maybe even unpin?)

Reply:
Since we only use two paasta scripts in this container that only communicate via exit code I think it's fairly safe to unpin, I've also added some comments to those scripts to be careful